superset/migrations/versions/6a8d1a21b532_.py

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""empty message
```

Comment:
Was this file added by error?

Reply:
no,this is due to 2 different migration heads merging present after I rebased this branch with latest master.
If this is not acceptable,I can make new migration on rebased branch,so only 1 file exists.

Reply:
Can you solve it by updating the parent revision ID in your migration script to the latest HEAD?

Reply:
fixed